pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/LiteralTransformFunction.java

```diff
@@ -269,4 +277,15 @@ public RoaringBitmap getNullBitmap(ValueBlock valueBlock) {
     bitmap.add(0L, length);
     return bitmap;
   }
+
+  /*
+   * Util function to check whether the Literal transform is null or not.
+   */
+  public static boolean isNullLiteralTransform(TransformFunction function) {
```

Comment:
Suggest adding  `boolean isNull()` instead. Checking `_dataType` itself should be enough (consistent with `getNullBitmap()`)